packages/arb-token-bridge-ui/src/util/orbitChainsList.ts

```diff
@@ -206,6 +206,49 @@ export const orbitMainnets: {
       }
     }
   },
+  70701: {
+    chainId: 70701,
+    confirmPeriodBlocks: 43200,
+    ethBridge: {
+      bridge: '0x53D82686BC9827fEc03bcEe661B37b855A18EcA9',
+      inbox: '0xEe30EfcaF812d10e1EFE25E9458f76a39DAD3239',
+      outbox: '0xa4270256B160C3Ebec2d6914a906c7EC38D8d072',
+      rollup: '0x330F8fEB25f3427cABA32446728C36ae67f2135b',
+      sequencerInbox: '0x6eE94AD8057Fd7Ba4d47bb6278a261c8a9FD4E3f'
+    },
+    explorerUrl: 'https://explorer-proofofplay-boss-mainnet.t.conduit.xyz',
+    rpcUrl: 'https://rpc.boss.proofofplay.com',
+    name: 'Proof of Play Boss',
+    slug: 'pop-boss',
+    parentChainId: 42161,
+    isCustom: true,
+    isTestnet: false,
+    tokenBridge: {
+      parentCustomGateway: '0x7D25Cc5b71AC932F696F3f4Da109e867395f4F05',
+      parentErc20Gateway: '0x298eb8d9f2F046AC60c01535fad40320CCdeB7c0',
```

Comment:
If this should be `standardGateway` this should be `0x410a210d51597be9e575F9b46c11eB865c0Ee073` ?